### DISCUSSION TITLE
Contest log remains empty while live contest logging

### DISCUSSION THREAD

**Discussion by dk5cr** [magicbug/Cloudlog#2918] [General]
Installed version is 2.6.1

I installed the current version and it shows a strange behaviour: 
Live contest logging with serial nr. and exchange schows me a worked stations as "worked before" in red, but the list at the bottom of the page remains empty. Is also is empty the whole time, no logged entries will be shown there. 
In the main list of QSO done the QSO are shown correctly with the attached parameters (serial/exchange). 

Is there a bug or did i missed some in the process of adding a new contest? 

Cheers and thanks 
Cornel, DK5CR 

PS: I didn't find a topic matching to this issue with the github search... If there is one please gimme a link to this. Thanks. 

**no entries on worked_before:**
![cloudlog_error_contest](https://github.com/magicbug/Cloudlog/assets/149412955/7d50e184-613a-4db6-a830-00a1460b7830)

**no entries at all:** 
![cloudlog_error_contest_no_entries](https://github.com/magicbug/Cloudlog/assets/149412955/fa0efed5-5827-49d6-bf1d-003b7ede1cac)

**Comment by magicbug** [comment]
Looks like this is a bug just tried to add a QSO and doesn't show for me either, will report back

**Reply by R1BLH** [reply]
It's about the same situation when you enter the log of past contests.
After saving the QSO data, it does not appear in the list, but it is entered in the general log. Refreshing the page does not help.

In addition, we need the ability to select the station location when entering the log of past contests. Now the log entry is made according to the active location of the station. Yesterday there were competitions on the work of their fields, for them a separate station location was created.

**Comment by magicbug** [comment]
It seems if you refresh the page after inserting the first QSO it starts to behave but somethings odd and needs resolved

**Comment by dk5cr** [comment]
Hi, just saw that you fixed the problem with worked-before stations in a contest. So i updated to 2.6.3, which tells me the fix in its greeting-window. But unfortunately the list is empty anyway and when i started a new contest session and entered a QSO, the registred callsign is shown after confirming the QSO in the proposed callsigns for the next entry. There is no "worked before" added when typing in the same callsign once again and tie list of logged signs is as empty as it was in the last version. 
I've got no idea, what goes wrong here and if there is something to to to make it working. Any ideas? 
Cheers and 73, DK5CR 

PS: Even refreshing the page doesn't work!

**Reply by magicbug** [reply]
You might need to fully clean out browser cache

**Reply by dk5cr** [reply]
Sorry, i did of course, but forgot to mention. 
When i entered a new callsign in contestmode the list at the bottom says: 
"No data available in table" despite the fact, that i entered at least one QSO. 

After submitting a new QSO the fields are empty and the hint "No data available in table" is hidden/invisible. 
By adding the same callsign i expected to show me the already finished QSO few minutes ago or at least show me "Worked before". But nothing happens. And yes, i checked, it is version 2.6.3., downloaded and installed half an hour ago. 

btw: Refreshing the page doesn't change the situation!

**Reply by dk5cr** [reply]
Doesn't also depend on the chosen contest. I started a new contest session with another contest (something with Spanish...) and typed in two different QSOs, but the same malfunction or better no function of the list and the worked-before. 
Sorry, guys, for me the solution doesn't work. 

I'm using firefox with a linux server in the same net: 
Server Software 	lighttpd/1.4.59
PHP Version 	7.4.33
MySQL Version 	5.5.5-10.5.21-MariaDB-0+deb11u1

Could this be a reason for the mentioned problems?

**Comment by R1BLH** [comment]
@magicbug Here is some additional information about issue with the contest logging.
Saturday was a contest on VHF, due to the fact that the contest session input functionality in Cloudlog (v2.6.3, dev branch) is now broken, I logged on paper.
Then I decided to transfer the records to Cloudlog to export to Cabrillo format, which can then be converted to EDI format (direct export to this report format is very missing).
The exchange in this contest included a report (RST), QSO sequence number, and grid square. I entered all of this when filling out the post contest data, and noted in the input format that all of these fields must be included.
![2024-02-18_11-28-30](https://github.com/magicbug/Cloudlog/assets/16667173/9e8313ba-2243-4b1f-8f3f-088761836889)

However, in Cabrillo, the gridsquare did not get gridsquare, did not get gridsquare, and in the log, the corresponding fields on the contest tab were empty.
![2024-02-18_11-29-16](https://github.com/magicbug/Cloudlog/assets/16667173/7aa0008c-dccb-4206-9ce2-cc2838a9d280)
![2024-02-18_11-32-04](https://github.com/magicbug/Cloudlog/assets/16667173/17273de7-ca31-4b90-b57c-a75ffaaa4e50)
![2024-02-18_11-29-59](https://github.com/magicbug/Cloudlog/assets/16667173/cf815450-9714-48a7-876b-a38c1c22a2a7)
![2024-02-18_11-30-23](https://github.com/magicbug/Cloudlog/assets/16667173/d64659b6-8dc7-44d8-aa1e-a6995908e7be)

However, if you manually edit the QSO in the log and enter the missing data, it will be saved and displayed in the log.

![2024-02-18_11-30-55](https://github.com/magicbug/Cloudlog/assets/16667173/a35992fc-5597-4415-aca2-28200a55c98c)
![2024-02-18_11-31-23](https://github.com/magicbug/Cloudlog/assets/16667173/3d4df9b2-bac2-46db-b9ef-333bf935a6f8)

**Reply by magicbug** [reply]
Thanks will dig into it
